.vitepress/config.mts

```diff
@@ -280,6 +280,7 @@ export default withMermaid(
                 { text: "Rewards", link: "rewards" },
                 { text: "Nodes Connection", link: "nodes-connection" },
                 { text: "Running a Node", link: "running-a-node" },
+                { text: "Running a Testnet Node", link: "running-a-sora-testnet-node" },
```

Comment:
```suggestion
                { text: "Running a SORA v3 Testnet Node", link: "running-a-sora-testnet-node" },
```

Reply:
Resolved in commit 88c29302a00fa18434ec2ba84d3c872148245122